src/constants/index.ts

```diff
@@ -918,6 +941,16 @@ export const POOLS_MAP: PoolsMap = {
     route: "arbusd",
     rewardPids: buildPids({ [ChainId.ARBITRUM]: 1 }),
   },
+  [FRAX_ARB_USD_POOL_V2_NAME]: {
+    name: FRAX_ARB_USD_POOL_V2_NAME,
+    addresses: FRAX_ARB_USD_SWAP_V2_ADDRESSES,
+    lpToken: FRAX_ARB_USD_SWAP_V2_TOKEN,
+    poolTokens: FRAX_ARB_USD_POOL_V2_TOKENS,
+    isSynthetic: false,
+    type: PoolTypes.USD,
+    route: "frax-arbusdv2",
+    rewardPids: buildPids({ [ChainId.ARBITRUM]: 1 }),
```

Comment:
double check with John on this, but I believe this pool is not yet in the arbitrum reward contract. I think it would be 2 (not 0-indexed) when added

Reply:
John says it's going to be managed by Frax team on staking.frax.finance.